egs/mls_english/ASR/local/utils/asr_datamodule.py

```
@@ -0,0 +1,253 @@
+    def load_dataset(self, dataset_path: Optional[str] = None) -> None:
+        """Load the HuggingFace dataset."""
+        dataset_path = dataset_path or self.args.dataset_path
+        logging.info(f"Loading MLS English dataset from: {dataset_path}")
+
+        try:
+            from datasets import load_dataset
+
+            self.dataset = load_dataset(dataset_path)
+            logging.info("Dataset loaded successfully")
+        except ImportError:
+            raise ImportError("Please install datasets package: pip install datasets")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)